src/test/regress/expected/multi_size_queries.out

```diff
@@ -99,6 +106,25 @@ SELECT citus_total_relation_size('customer_copy_hash');
                    2646016
 (1 row)
 
+-- Tests on index on distributed table.
+SELECT citus_table_size('customer_copy_hash_pkey');
+ citus_table_size
+---------------------------------------------------------------------
+          1048576
+(1 row)
+
+SELECT citus_relation_size('customer_copy_hash_pkey');
+ citus_relation_size
+---------------------------------------------------------------------
+             1048576
+(1 row)
+
+SELECT citus_total_relation_size('customer_copy_hash_pkey');
+ citus_total_relation_size
+---------------------------------------------------------------------
+                   1048576
+(1 row)
+
```

Comment:
Can we add a test for below scenario?:

There is a distributed & partitioned table, say p, that has two child rels, say p_c1 & p_c2; where p has an index named as p_idx and children do so (implicitly) too, as in p_c1_idx & p_c2_idx.

And we should make sure that following always returns true:
```sql
SELECT citus_relation_size(p_idx) <
       citus_relation_size(p_idx) + citus_relation_size(p_c1_idx) + citus_relation_size(p_c2_idx);
```

This is to make sure that the following comment holds for indexes too:
```c
	/*
	 * We pass false here, because if we optimize this, we would include child tables.
	 * But citus size functions shouldn't include them, like PG.
	 */
	bool optimizePartitionCalculations = false;
```

Reply:
You got it. I though about this scenario but didn't considered too much as a problem, given the code.
Will update but with real values, not sure to see the point of the boolean test.

Reply:
Looks like I made a mistake, I meant:

```sql
SELECT citus_relation_size(p_idx) <
       citus_relation_size(p_c1_idx) + citus_relation_size(p_c2_idx);
```

Reply:
added more tests